solr/core/src/test/org/apache/solr/search/TestSolr4Spatial2.java

```diff
@@ -120,21 +120,27 @@ public void testRptWithGeometryGeo3dField() throws Exception {
   
   @Test
   public void testLatLonRetrieval() throws Exception {
-    assertU(adoc("id", "0",
-        "llp_1_dv_st", "-75,41",
-        "llp_1_dv", "-80.0,20.0",
-        "llp_1_dv_dvasst", "40.299599,-74.082728"));
+    assertU(adoc("id", "0", "llp_1_dv_st", "-75,41")); // stored
```

Comment:
Please also test -90 and +90, -180 and +180.  This will help ensure there aren't edge cases (literally) in the DV decoding logic.

Reply:
Sure. will add